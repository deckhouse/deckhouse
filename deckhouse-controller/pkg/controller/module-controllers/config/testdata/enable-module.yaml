---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleConfig
metadata:
  name: test-module
spec:
  enabled: true
  updatePolicy: Auto
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: test-module
properties:
  availableSources:
    - test-source
status:
  phase: Available
  conditions:
    - type: EnabledByModuleConfig
      status: "False" 