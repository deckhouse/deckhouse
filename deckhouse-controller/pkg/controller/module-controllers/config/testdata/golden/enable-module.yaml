---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleConfig
metadata:
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/module-registered
  name: test-module
  resourceVersion: "1001"
spec:
  enabled: true
  updatePolicy: Auto
status:
  message: ""
  version: "1"
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  creationTimestamp: null
  name: test-module
  resourceVersion: "1001"
properties:
  availableSources:
  - test-source
  source: test-source
  updatePolicy: Auto
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: null
    status: "True"
    type: EnabledByModuleConfig
  phase: Available
